src/core/trulens/core/database/migrations/versions/10_create_event_table.py

```diff
@@ -0,0 +1,60 @@
+"""create event table
```

Comment:
@sfc-gh-dkurokawa is there a way to make this optional for users? Though I don't foresee changes, this is still an incomplete feature so there might be unexpected changes to the db schema

Reply:
I think we can just have an experimental flag that sets the revision to upgrade to when we call `db_migrations.upgrade_db`. That is, if we're doing this OTEL stuff we can do this event table, otherwise we can do the regular thing for the time being.